cookbook/entities/deleting_tasks.rst

```diff
@@ -0,0 +1,113 @@
+How to Delete Tasks
```

Comment:
This file describes how to configure deletion only for the grid. So, i think you should specify this at the header.